.pre-commit-config.yaml

```diff
@@ -22,7 +22,7 @@ repos:
       - id: check-merge-conflict
       - id: check-yaml
         args: ["--allow-multiple-documents"]
-      - id: detect-private-key
+      #- id: detect-private-key
```

Comment:
We should just delete this. We're not going to use it. It doesn't work as easily as I originally thought.
```suggestion
```